### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by modery**
Correcting some wrong product name spellings

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/azure-integration.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ For more information about Microsoft 365 and Azure AD, see [Microsoft 365 identi
...
-|PowerApps  <br/> | Power apps are focused apps for mobile devices that can connect to your existing data sources like SharePoint lists and other data apps. See [Create a PowerApp for a list in SharePoint Online](https://support.office.com/article/9338b2d2-67ac-4b81-8e67-97da27e5e9ab) and the [PowerApps page](https://powerapps.microsoft.com/) for details.  <br/> |
+|Power Apps  <br/> | Power Apps are focused apps for mobile devices that can connect to your existing data sources like SharePoint lists and other data apps. See [Create a PowerApp for a list in SharePoint Online](https://support.office.com/article/9338b2d2-67ac-4b81-8e67-97da27e5e9ab) and the [Power Apps page](https://powerapps.microsoft.com/) for details.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/azure-integration.md]:
```suggestion
|Power Apps  <br/> | Power Apps are focused apps for mobile devices that can connect to your existing data sources like SharePoint lists and other data apps. See [Create a Power App for a list in SharePoint Online](https://support.office.com/article/9338b2d2-67ac-4b81-8e67-97da27e5e9ab) and the [Power Apps page](https://powerapps.microsoft.com/) for details.  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @modery, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @modery

**Pull request merged**